auditbeat/module/file_integrity/action.go

```diff
@@ -83,6 +83,9 @@ func (action Action) String() string {
 	return strings.Join(list, "|")
 }
 
+// MarshalText marshals the Action to a string.
```

Comment:
Minor details but it marshals it to a string and converts it then to a byte array :-)